maestro-cli/src/main/java/maestro/cli/runner/FileWatcher.kt

```diff
@@ -0,0 +1,54 @@
+package maestro.cli.runner
+
+import com.sun.nio.file.SensitivityWatchEventModifier
+import java.io.File
+import java.nio.file.FileSystems
+import java.nio.file.Path
+import java.nio.file.StandardWatchEventKinds
+import java.nio.file.WatchKey
+
+class FileWatcher {
```

Comment:
@Leland-Takamine should this be an `object` too perhaps?

Reply:
This class holds state so it needs to be an instantiable class